app/views/layouts/landingpage.html.haml

```diff
@@ -11,7 +11,7 @@
     %meta{:content => meta_title, :property => "og:title"}/
     %meta{:content => "website", :property => "og:type"}/
     %meta{:content => url_for(:only_path => false), :property => "og:url"}/
-    %meta{:content => asset_url("open-graph-logo.png"), :property => "og:image"}/
+    %meta{:content => asset_url("open-graph-logo.png", skip_pipeline: true), :property => "og:image"}/
```

Comment:
Das kenn ich gar nicht, was macht das?

Reply:
Das habe ich so gelöst, wegen: https://github.com/rails/rails/pull/26226. Rails wirft jetzt einen Fehler, wenn es Assets nicht in der Pipeline findet:
> The asset "open-graph-logo.png" is not present in the asset pipeline.

Um diesen Fehler zu vermeiden, wird hier die Asset-Pipeline nicht verwendet. (Das Bild liegt in `public/`)